{{ cookiecutter.package_name }}/.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,84 @@
+# GitHub Actions workflow for testing and continuous integration.
+#
+# This file performs testing using tox and tox.ini to define and configure the test environments.
+
+name: CI Tests
+
+on:
+  push:
+    branches:
+    - master
+    tags:
+  pull_request:
+
+jobs:
+  # Github Actions supports ubuntu, windows, and macos virtual environments:
+  # https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+  ci_tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - name: Code style checks
```

Comment:
If the code style check fails then no other test is run. I implement a fix for this in the example file I included in https://github.com/astropy/package-template/pull/449#issuecomment-729160415

Reply:
as i said below, i think having code style checks fail and stop everything is appropriate when starting with a fresh code base. again, this is a use case where better upstream support is required: https://github.com/actions/toolkit/issues/399. both your workaround and using `continue-on-error` cause the code style check to still show up as a failure that would still block merging. i would advocate against implementing those workarounds here and would rather wait for proper `allow-failure` support.